### DISCUSSION TITLE
Start Service via RTR

### DISCUSSION THREAD

**Discussion by lcagy** [CrowdStrike/psfalcon#361] [Q&A]
Hi All.. 

I´m trying start a windows service using API. 

try this script. 

```powershell
#Requires -Version 5.1
using module @{ModuleName='PSFalcon';ModuleVersion ='2.2'}
Request-FalconToken -ClientId @@@@  -ClientSecret #####  -Cloud us-1

Find-FalconHostname -Path .\hostname.txt | Invoke-FalconRtr -Command runscript  -Argument'-CloudFile="Start_Umbrella"' -QueueOffline $true
``` 

I received this message

Write-Result: C:\XXX\PowerShell\Modules\PSFalcon\2.2.5\private\Private.ps1:627
Line |
 627 |                  Write-Result $Object
     |                  ~~~~~~~~~~~~~~~~~~~~
     | [{"code":403,"message":"access denied, authorization failed"}]

aid                              session_id                           offline_queued
---                              ----------                           --------------
5eee891992a740bc8665b9e7d31e5056 710090d9-ecc4-4ba8-9b09-59f663e25188          False

when check in host, the service not started

**Comment by bk-cs** [comment]
The syntax looks correct. My guess is either that your API client failed the initial token request (IP allowlisting enabled?) or that it doesn't have the proper RTR permission.
